src/content/guides/partial-payment-extension.mdx

```diff
@@ -46,7 +46,7 @@ autonumber
 A patron asks the merchant to process a specified part of the total basket
 amount using Centrapay. This could be due to the patron knowing their Centrapay
 Asset balance wouldn’t cover the complete basket amount or enable them to use a
-set amount of funds from a fixed value asset eg. Gift Card to complete the payment.
```

Comment:
Can still pay with giftcard I wonder how we should word this or token is ok?

Reply:
I'm easy, it's just an example so Token is fine with me. We still have a couple of references to gift cards elsewhere